### ISSUE TITLE
maxLines does not work with markupEnabled

### ISSUE THREAD

**Issue by cpetrov** [eclipsesource/tabris-js#1927]
### Problem description

When `maxLines` and `markupEnabled` are simultaneously set, `maxLines` does not have any effect. 

### Environment

- Tabris.js version: 2.8.1
- OS: iOS 12.4

### Code snippet

```javascript
const {TextView, ui} = require('tabris');

// Create text views with different alignments

new TextView({
  left: 10, top: 10, right: 10, maxLines: 1, markupEnabled: true,
  text: 'Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.',
  alignment: 'left'
}).appendTo(ui.contentView);

```

**Comment by mpost** [comment]
Relates to https://github.com/eclipsesource/tabris-js/issues/1867

**Comment by patrykmol** [comment]
@cpetrov I cannot reproduce this issue in latest nightly.

**Comment by cpetrov** [comment]
👍 @patrykmol my mistake, it affects only 2.8.0 and is already fixed in 2.8.1.
